apps/modernization-ui/src/apps/page-builder/page/management/edit/subsection/Subsection.tsx

```diff
@@ -76,8 +76,7 @@ export const Subsection = ({
     return (
         <div className={styles.subsection}>
             <SubsectionHeader
-                name={subsection.name ?? ''}
-                id={subsection.id}
+                subsection={subsection}
                 questionCount={subsection.questions?.length ?? 0}
```

Comment:
Suggestion: if changing this component to accept the entire `subsection`, then the `questionCount` prop can be removed and calculated within the component